ambry-router/src/main/java/com.github.ambry.router/DefaultKeyManagementService.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.router;
+
+import com.github.ambry.account.Account;
+import com.github.ambry.account.Container;
+import com.github.ambry.config.KMSConfig;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import javax.crypto.spec.SecretKeySpec;
+import org.bouncycastle.util.encoders.Hex;
+
+
+/**
+ * Default {@link KeyManagementService} which returns a default key {@link SecretKeySpec} for any
+ * {@link #getKey(short, short)} calls.
+ */
+public class DefaultKeyManagementService implements KeyManagementService<SecretKeySpec> {
```

Comment:
it would be nice if we can get rid of the `Default` in the name. Maybe name it something like `SingleKeyService` (or something that describes the impl better). `Default` - 1) Need not be default in the future 2) Doesn't provide any description about impl

Reply:
Will fix it